docs/content.zh/docs/connectors/table/hive/hive_functions.md

```diff
@@ -73,6 +73,34 @@ Some Hive built-in functions in older versions have [thread safety issues](https
 We recommend users patch their own Hive to fix them.
 {{< /hint >}}
 
+## Use Native Hive Aggregate Functions
+
+If [HiveModule]({{< ref "docs/dev/table/modules" >}}#hivemodule) is loaded with a higher priority than CoreModule, Flink will try to use the Hive built-in function first. And then for Hive built-in aggregation function,
+Flink currently uses sort-based aggregation strategy. Compared to hash-based aggregation strategy, the performance is one to two times worse, so from Flink 1.17, we have implemented some of Hive's aggregation functions natively in Flink.
+These functions will use the hash-agg strategy to improve performance. Currently, only five functions are supported, namely sum/count/avg/min/max, and more aggregation functions will be supported in the future.
```

Comment:
for the fix length agg buffer, we can use hash-agg, otherwise sort-agg will be chosed

Another performance improvement is code gen